crates/graphql-schema-diff/src/patch/schema_definitions.rs

```diff
@@ -14,7 +14,12 @@ pub(super) fn patch_schema_definition<T: AsRef<str>>(
     let mut new_mutation_type = None;
     let mut new_subscription_type = None;
 
-    for change in paths.iter_exact([""; 3]) {
+    let prefix = match definition_or_extension {
+        DefinitionOrExtension::Extension => ":schema[0]",
```

Comment:
This is of course not correct in general, but it's no less correct than what we had before. Extensions support in graphql-schema-diff has to be implemented, and the first step is to represent extensions in paths like as done in this PR.